api/app/routers/playthroughs.py

```
@@ -0,0 +1,266 @@
+            offset=offset,
+        )
+    except ValidationError as e:
+        raise HTTPException(status_code=422, detail=str(e))
+
+
+@router.post(
+    "", response_model=PlaythroughResponse, status_code=status.HTTP_201_CREATED
+)
+async def create_playthrough(
+    playthrough_data: PlaythroughCreate,
+    current_user: CurrentUser = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)